src/components/Author/index.tsx

```diff
@@ -46,22 +25,17 @@ const Author = ({
   const mleft = index === 0 ? { marginLeft: 0 } : {};
 
   return (
-    <li css={list}>
+    <li className="list">
       <a
-        css={link}
+        className="link"
         href={githubLink}
         title={username}
         key={username}
         target="_blank"
         rel="noopener noreferrer"
         style={mleft}
       >
-        <img
-          css={img}
-          className="author-img"
-          src={githubImgLink}
-          alt={username}
-        />
+        <img className="img " src={githubImgLink} alt={username} />
```

Comment:
I think this is a good move to standardize to SCSS 👍  Small space after the 'g' of img